webpack.config.js

```diff
@@ -147,7 +147,7 @@ var config = {
     module: {
         rules: [
             {
-                test: /\.tsx?$/,
+                test: /\.(tsx|ts)?$/,
```

Comment:
the question mark doesnt make sense anymore I think..

Reply:
Change was reverted.